lib/SyTest/HomeserverFactory/Synapse.pm

```diff
@@ -103,6 +108,11 @@ sub _init
    $self->{args}{redis_host} = "";
 }
 
+sub implementation_name
+{
+   return "synapse";
+}
+
```

Comment:
this should be unnecessary, as it will be inherited from `SyTest::HomeserverFactory::Synapse` ?